.circleci/config.yml

```diff
@@ -751,6 +748,28 @@ parameters:
 workflows:
   version: 2
 
+  build-pr:
+    # Launched on PR commits
+    jobs:
+      - app-build-and-store-job:
+          <<: *filter-pr-branch
+          name: << matrix.study_key >>-build
+          matrix:
+            alias: build-app
+            parameters:
+              study_key: [ basil, osteo, angio, brain, mbc, testboston, mpc, atcp, rarex, rgp, esc, prion, cgc, circadia, pancan, singular, brugada, lms, fon ]
+      - app-ui-unit-test:
+          <<: *filter-pr-branch
+          name: << matrix.study_key >>-unit-test
+          matrix:
+            alias: ui-unit-test
+            parameters:
+              study_key: [ basil, osteo, angio, brain, mbc, testboston, mpc, atcp, rarex, rgp, esc, prion, cgc, circadia, pancan, singular, brugada, lms, fon ]
+      - toolkit-sdk-unit-test:
+          <<: *filter-pr-branch
+      - playwright-build:
+          <<: *filter-pr-branch
```

Comment:
This new workflow is automatic when new commits are pushed to github. It's not manually triggered by `run_ci.sh` to CircleCI API call.

- Build and lint all studies
- Run unit tests for all studies
- Run toolkit and ddp-sdk unit tests
- Build and lint playwright-e2e subproject

Remaining question: Is there any study which we can skip build and lint and run unit tests? Right now, all studies are included.